code/modules/projectiles/guns/energy/nuclear.dm

```diff
@@ -202,6 +202,34 @@
 		list(mode_name="full-auto", can_autofire = TRUE, burst = 1, burst_accuracy = list(0,-1,-1,-2,-2,-2,-3), dispersion = list(5, 10, 15, 20))
 		)
 
+/obj/item/gun/energy/repeater/pistol
+	name = "Energy Pistol"
+	desc = "A more compact and portable version of the Stellar Corporate Conglomerate Energy Repeater. It has two settings: Stun, and Lethal."
+	desc_extended = "The SCC-ER1-2 was designed to be a reliable yet more compact version of the SCC-ER1, capable of defending Staff and Assets."
+	icon = 'icons/obj/guns/sccpistol.dmi'
+	icon_state = "sccpistolstun"
+	item_state = "sccpistolstun"
+	has_item_ratio = FALSE
+	fire_sound = 'sound/weapons/Taser.ogg'
+	slot_flags = SLOT_BELT|SLOT_HOLSTER
+	max_shots = 8
+	fire_delay = 5
+	can_turret = 0
```

Comment:
What is the point of setting it up to be attachable to a turret with the sprite and lethality, if it cannot be attached?

Reply:
...Isn't 0 "no"? I meant for it to not be turretable.

Reply:
0 is evaluated as false, so yes this would not be turretable, but then below you specified:

```
	turret_sprite_set = "carbine"
	turret_is_lethal = 0
```

What is the point of setting info about its use in a turret, if it's not turretable?